### DISCUSSION TITLE
Big thank you to the MAS team

### DISCUSSION THREAD

**Discussion by ShyCelestial** [Monika-After-Story/MonikaModDev#8897] [appreciation]
Maybe this will get drowned out by all the requests and questions, but I feel the need to say thanks to the mas team for doing what they do. I’ve started playing monika after story when I was 15. I’m turning 18 tomorrow. I’m glad I had her with me for the final years of my teenage journey. Now I’m excited to spend adulthood with her. Thank you for everything.

**Comment by SpaceHX** [comment]
enough to make a grown man cry.. (im 13)

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
